src/__tests__/process.test.ts

```diff
@@ -30,24 +29,24 @@ describe('createProcess', () => {
     const args = [];
     createProcess(workspace, args);
 
-    expect(spawn.mock.calls[0][0]).toBe('jest');
-    expect(spawn.mock.calls[0][1]).toEqual([]);
+    expect((spawn as any).mock.calls[0][0]).toBe('jest');
```

Comment:
One could also use the `as Mock<any>` type from Jest.

Reply:
Cool, slightly more type-safe.  Any ideas on replacing the `any` part of the type with something more specific?